eng/dtdl/device-name-based-operations.json

```diff
@@ -55,12 +55,18 @@
           "@type": "Object",
           "fields": [
             {
-              "@type": ["Field", "Required"],
+              "@type": [
```

Comment:
Are these formatting changes intentional? Here and in the cargo.toml

Reply:
i copy pasted content - did not do any format.

Reply:
Raul said should be fine - some auto formatting going on save